python/ray/autoscaler/updater.py

```diff
@@ -232,8 +242,10 @@ def run(self,
             timeout=120,
             exit_on_fail=False,
             port_forward=None,
-            with_output=False):
-
+            with_output=False,
+            ssh_options=None):
```

Comment:
maybe raise assertionerror is ssh_options is not proper type?

i think this is easily confusable

Reply:
Sounds good!